dirsrvtests/tests/suites/retrocl/retrocl_indexing_test.py

```diff
@@ -69,6 +83,76 @@ def test_indexing_is_online(topo):
     assert not topo.standalone.searchAccessLog('Partially Unindexed Filter')
 
 
+def test_reindexing_retrochangelog(topology_m1):
+    """ Test that the retro changelog can be successfully reindexed
+
+        :id: 26d8df59-0886-4098-8d2c-b9337accc908
+        :setup: Supplier instance
+        :steps:
+            1. Enable retro changelog and restart instance
+            2. Create test entries
+            3. Create index
+            4. Reindex retro changelog
+            5. Verify indexed search
+            6. Clean up test entries
+        :expectedresults:
+            1. Success
+            2. Success
+            3. Success
+            4. Success
+            5. Success
+            6. Success
+        """
+
+    log.info('Enable retro changelog and restart instance')
+    retrocl = RetroChangelogPlugin(topology_m1.ms["supplier1"])
+    retrocl.enable()
+    topology_m1.ms["supplier1"].restart()
+
+    log.info('Creating test entries')
+    for count in range(MAX_OTHERS):
+        name = "{}{}".format(OTHER_NAME, count)
+        topology_m1.ms["supplier1"].add_s(
+            Entry(("cn={},{}".format(name, DEFAULT_SUFFIX), {
+            'objectclass': "top person".split(),
+            'sn': name,
+            'cn': name})),
+            escapehatch='i am sure')
+
+    log.info('Test entries created')
+
+    try:
+        log.info('Verify number of created entries')
+        check_entries = topology_m1.ms["supplier1"].search_s(
+            RETROCL_SUFFIX, ldap.SCOPE_ONELEVEL, "(objectclass=*)", escapehatch='i am sure')
+        assert len(check_entries) == MAX_OTHERS
+
+        log.info('Create index')
+        args = {INDEX_TYPE: 'eq'}
+        for attr in ATTRIBUTES:
+            topology_m1.ms["supplier1"].index.create(suffix=RETROCL_SUFFIX, attr=attr, args=args)
+
+        topology_m1.ms["supplier1"].restart()
+
+        log.info('Reindex retro changelog')
+        args = {TASK_WAIT: True}
+        for attr in ATTRIBUTES:
+            rc = topology_m1.ms["supplier1"].tasks.reindex(suffix=RETROCL_SUFFIX, attrname=attr, args=args)
+            log.info('Verify reindexing task was successful for attribute {}'.format(attr))
+            assert rc == 0
+
+        log.info('Check reindexed search')
+        for attr in ATTRIBUTES:
+            ents = topology_m1.ms["supplier1"].search_s(RETROCL_SUFFIX, ldap.SCOPE_SUBTREE, "({}=hello)".format(attr),
+                                                        escapehatch='i am sure')
+            assert len(ents) == 0
+    finally:
+        log.info('Clean up the test entries')
+        entries = topology_m1.ms["supplier1"].search_s(DEFAULT_SUFFIX, ldap.SCOPE_SUBTREE, "(objectClass=person)", escapehatch='i am sure')
+        for entry in entries:
+            topology_m1.ms["supplier1"].delete_s(entry['dn'], escapehatch='i am sure')
```

Comment:
And as `UserAccounts` will be used, the same object instance can be used for the removal here.